services/bettersocial/post/createPostV3.js

```diff
@@ -330,7 +330,7 @@ const BetterSocialCreatePostV3 = async (req, isAnonimous = true) => {
     else post = await Getstream.feed.createPost(req?.token, data);
 
     // Upvote post before sending the success message
-    await upVote(post?.id, req.token, req?.userId);
+    await upVote(post?.id, req.token, userDetail?.user_id);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `upVote` function is now being called with `userDetail?.user_id` instead of `req?.userId`. Ensure that `userDetail` is defined and contains the `user_id` property before this line. If not, it could lead to unexpected behavior or errors. Consider adding error handling or a null check for `userDetail`.

```diff
-    await upVote(post?.id, req.token, userDetail?.user_id);
+    if (userDetail && userDetail.user_id) {
+        await upVote(post?.id, req.token, userDetail.user_id);
+    } else {
+        // Handle error or fallback scenario here
+    }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->